content/webapp/components/PhysicalItems/PhysicalItems.tsx

```diff
@@ -73,8 +73,8 @@ const PhysicalItems: FunctionComponent<Props> = ({
   const [itemsState, setItemsState] = useItemsState(workItems);
 
   useEffect(() => {
-    setPhysicalItems(workItems);
-  }, [workItems]);
+    setPhysicalItems(physicalItems);
+  }, [physicalItems]);
```

Comment:
What does this do differently than before? The `useEffect` would get triggered when `physicalItems` changes, which I feel would be a loop effect but I might be confusing things.